README.md

```diff
@@ -115,6 +119,32 @@ Course = {
   name: string
 }
 ```
+
+### Course info by name
+
+return
+
+```
+  await codio.v1.course.findOneByName(courseName, withHiddenAssignments)
+```
+return `CourseWithModules` object
+```
+CourseWithModules = {
```

Comment:
can we keep using `Course`?

Reply:
Old method from octopus do not return course, it returns Module[]

Reply:
`course.info` returning `Modules[]` doesn't make any sense